code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -668,4 +731,3 @@ s
 #undef LAYING_EGGS
 #undef MOVING_TO_TARGET
 #undef SPINNING_COCOON
-#undef MAX_WEBS_PER_TILE
```

Comment:
if this needs to be used in other files it should go in __DEFINES

Reply:
:+1:

Reply:
This is done now